### ISSUE TITLE
Graphical bug when highlighting an icon that overlaps the scroll bar

### ISSUE THREAD

**Issue by CPCTC** [Tomas-M/xlunch#137]
When an icon overlaps the scroll bar, the scroll bar gets a little messed up when the icon gets highlighted. The best way to reproduce this is to start xlunch in windowed mode, resize it (decrease width), then mouse over the rightmost icons.

Screenshots:

![1](https://user-images.githubusercontent.com/63987003/149442990-0195ab3d-b72d-4c33-92e3-60c0bc6e3606.png)
![2](https://user-images.githubusercontent.com/63987003/149442991-7c6fcf30-5e07-423a-9dbe-b40ce8338300.png)
xlunch was invoked with `xlunch --windowed < entries.dsv`

**Comment by CPCTC** [comment]
This can also be triggered regardless of window width when using `--textafter`, which is how I first discovered this.

**Comment by PMunch** [comment]
The bug is because the icons overlap the scrollbar, if you set the right border to be big enough to cover the scrollbar icons will flow to the next line instead of overlapping. I guess @Tomas-M forgot to update the layouting when he added the scrollbar. The whole feature is a bit under-developed compared to everything else in xlunch. You can't even hide (well you can by setting it's opacity to full) or decide where to place it..

**Comment by Tomas-M** [comment]
Yeah the scrollbar was just a quick hack for my particular use case.
Patches to improve its customizability are more than welcome :)
